server/celery-queue/Dockerfile

```diff
@@ -6,12 +6,15 @@ ENV CELERY_BROKER_URL redis://redis:6379/0
 ENV CELERY_RESULT_BACKEND redis://redis:6379/0
 ENV C_FORCE_ROOT true
 ENV WEB_DOCKER_URL web
+ENV PATH=$PATH:~/.planutils/bin
 
 COPY . /queue
 WORKDIR /queue
 
-RUN pip install -r requirements.txt
-
+# Grabbing downward and lama
+RUN planutils install -f downward
```

Comment:
No need to do `downward`. Just `lama-first` (it has the right dependency to get `downward` in there).

Reply:
Gotcha.